strings/crustacean_replacement.json

```
@@ -0,0 +1,115 @@
+{
+
+    "crustacean": {
+		"simp": "shrimp",
```

```suggestion
	"crustacean": {
		"simp": "shrimp",
```
consistent formatting